adapters/mqtt-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -1176,6 +1180,7 @@ protected final void registerHandlers() {
             endpoint.publishAcknowledgeHandler(this::handlePubAck);
             endpoint.subscribeHandler(this::onSubscribe);
             endpoint.unsubscribeHandler(this::onUnsubscribe);
+            endpoint.pingHandler(v -> endpoint.pong());
```

Comment:
My understanding is that it is not necessary to register a ping handler because the vertx code already responds to pings automatically. Any reason you want to do this?

Reply:
True there is no actual need for this, I remove it.